package-res/resources/web/vizapi/DataTable.js

```diff
@@ -15,1094 +15,1220 @@
 *
 */
 
-/*
-pentaho.DataTable
-pentaho.DataView
+pentaho = typeof pentaho == "undefined" ? {} : pentaho;
 
-Client-side non-visual data tables.
+/**
+ * Pentaho JavaScript Visualization Data APIs.
+ *
+ * This module defines interfaces and classes
+ * used by visualizations for data representation and manipulatation.
+ *
+ * @module common-ui.vizapi.data
+ * @main common-ui.vizapi.data
+ */
+
+/**
+ * An _AbstractDataTable_ represents a set of tabular data.
+ *
+ * @class AbstractDataTable
+ * @constructor
+ */
+ pentaho.AbstractDataTable = function AbstractDataTable() {};
+
+/**
+ * Gets a column object, given its index.
+ *
+ * @method _getColumn
+ * @param {Number} colIdx The column index (zero-based).
+ * @return {Object} The column object.
+ * @protected
+ */
+
+/**
+ * Gets a row object, given its index.
+ *
+ * @method _getRow
+ * @param {Number} rowIdx The row index (zero-based).
+ * @return {Object} The row object.
+ * @protected
+ */
+
+/**
+ * Gets a cell, given its row and column index.
+ *
+ * @method _getCell
+ * @param {Number} rowIdx The cell's row index (zero-based).
+ * @param {Number} colIdx The cell's column index (zero-based).
+ * @return {Null|Object} The cell.
+ * @protected
+ */
+
+/**
+ * Gets the number of columns.
+ *
+ * @method getNumberOfColumns
+ * @return {Number} The number of columns.
+ */
+
+/**
+ * Gets the number of rows.
+ *
+ * @method getNumberOfRows
+ * @return {Number} The number of rows.
+ */
+
+// TODO: Not publicizing the add method yet, as it is not unit-tested,
```

Comment:
Then we both attribute the same meaning to TODO. We're just differing in the _when_ things need to be done :-)
However, I agree that an ATTENTION or NOTE is enough in this case. Will change.
Thanks